### PR TITLE
feat(experiments): Consolidate create experiment to one form

### PR DESCRIPTION
**Pull request by danielbachhuber**
## Changes

Consolidates the experiment creation UX to a form on one page:

CleanShot 2024-11-07 at 03 36 05@2x

A new experiment defaults to no defined goals, and the launch button is disabled until goals are defined:

CleanShot 2024-11-07 at 03 37 11@2x

Once the user defines a goal, they can launch the experiment:

CleanShot 2024-11-07 at 03 40 35@2x

## How did you test this code?

I created a new experiment with all of the fields filled out and verified the data saved as expected.

I verified the launch button was disabled when no goals were specified. I then verified the launch button could be used when a goal was identified.

### CODE REVIEW

**Comment by jurajmajerik** [comment]
Done :)
https://www.loom.com/share/e50901b48c8747838eeb0d152f3d9219?sid=40e979dd-7462-426b-ad1c-0f744255747b

(thanks for adding the tests!)

**Review by jurajmajerik — APPROVED** [review]

File: ee/clickhouse/views/test/test_clickhouse_experiments.py [hunk 1]
```diff
@@ -681,30 +681,13 @@ def test_invalid_create(self):
                 "end_date": None,
```
Comment (danielbachhuber) [inline ee/clickhouse/views/test/test_clickhouse_experiments.py:681]: @jurajmajerik Ah, I missed this file. Should the tests I created live here instead?

Also, out of curiosity, what's the reason 'clickhouse' is included in the file name?

Comment (jurajmajerik) [reply ee/clickhouse/views/test/test_clickhouse_experiments.py:681]: > Should the tests I created live here instead?

Ah good point. Yes indeed - do you mind moving them?

Comment (danielbachhuber) [reply ee/clickhouse/views/test/test_clickhouse_experiments.py:681]: > Yes indeed - do you mind moving them?

Yep, done in ea46fe8f4d

Comment (neilkakkar) [reply ee/clickhouse/views/test/test_clickhouse_experiments.py:681]: a long time ago, `ee` was very roughly for clickhouse backed stuff, and `posthog/` was for postgres backed stuff (when we had analytics data in postgres, and no support for open source clickhouse). Experiments were clickhouse only from the beginning, and thus put in ee like so.

**Pull request merged**